dev/Tiles/tile.h

```diff
@@ -0,0 +1,70 @@
+#include <stdbool.h>
+#include <stdlib.h>
+
+/**
+ * @param unsigned char id;
+ * @param bool collidable;
+ * @param bool flipX;
+ * @param bool flipY;
+ * @param bool useSpritePalette;
+ * @param bool priority;
+ * @param unsigned int computedId;
+*/
+typedef struct Tile
+{
+    unsigned char id;
+    bool collidable;
+    bool flipX;
+    bool flipY;
+    bool useSpritePalette;
+    bool priority;
```

Comment:
If you're concerned about RAM utilisation for many Tile objects you could use bit flags here although use of `bool`s is simpler and more readable so if space isn't an issue the way you've done it is better imo.